utils/cliff.toml

```diff
@@ -54,7 +54,10 @@ body = """
         {%- if commit.scope -%}
         {% else -%}
             - {{ commit.message | upper_first }}\
-                {{ self::render_commit_body(commit=commit) }}\
+                {% if commit.body is matching("Changelog: Title") %}
```

Comment:
I still want the third option to use the `Changelog:` footer contents instead.

Reply:
> afaict that's a feature request not a review 🤨

I fully agree. My "review" was just part of the momentary game we played that day/night. :wink: As I explained on Slack, my reason for not approving was that you said you didn't test it.